CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ Release channels have their own copy of this changelog:
 * [stable - v2.1](https://github.com/anza-xyz/agave/blob/v2.1/CHANGELOG.md)
 
 <a name="edge-channel"></a>
```

Comment:
is 2.3 still supposed to point to edge?

Reply:
In previous release, it seems we landed the "update changelog" PR prior to branching. Ie, https://github.com/anza-xyz/agave/pull/6424 came in after we cut v2.3 if I'm not mistaken.

So, I think just BP'ing 6424 would keep us consistent with past releases

Reply:
Yeah, I could see just deleting the channel indications as `v2.3` does go from `beta` to `stable` at some point, and keeping it noted here is just another thing to ~~remember to update~~ get out of date.

Reply:
@willhickey any guidance here?